components/assets/src/gltf/tinygltf.h

```diff
@@ -0,0 +1,44 @@
+#pragma once
```

Comment:
What exactly is this file for? The naming got me a bit confused in the first place.